tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, name, init_config, instances):
         self.timeout = float(instance.get('timeout', 10))
         self.collect_response_time = instance.get('collect_response_time', False)
         self.host = instance.get('host', None)
+        self.ip_cache_duration = int(instance.get('ip_cache_duration', 0)) or None
```

Comment:
```suggestion
        self.ip_cache_duration = int(instance.get('ip_cache_duration', 0)) or  CACHE_NEVER_EXIPRES
```
and then define `CACHE_NEVER_EXIPRES = None` somewhere at the top, it makes it more readable